js/ad_manager_vast.js

```diff
@@ -561,7 +561,14 @@ OO.Ads.manager(function(_, $) {
           viewMode,
           creativeData = {};
 
+    if (_isVpaidAd(currentAd)) {
       currentAd.data = currentAd.ad.data;
+
+      var mediaFileUrl = _cleanString(currentAd.ad.mediaFile.url);
```

Comment:
Sure well _isVPaidAd already checks for ad && ad.ad && ad.ad.data, but will make sure to check for mediaFile, which is mostly only for testing purposes, because basically we would never get here if mediaFile is empty/null, since this function is only loaded when `script.onload=initialize` otherwise if empty we would go to `script.onerror`